apcd-cms/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM taccwma/core-cms:ff737a3
```

Comment:
**Up-to-Date?** How about, instead of this Oct 17th commit of https://github.com/TACC/Core-CMS/pull/558/commits (not even the last commit of that PR), we use the commit in `main` ([list](https://github.com/TACC/Core-CMS/commits/main)) when that PR got merged i.e. `abd48a2`?